src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/Policies/EnumConverter.cs

```diff
@@ -9,12 +9,16 @@ internal class EnumConverter
         //todo: support [Flags]?
         public static bool TryGetFromJson(ReadOnlySpan<byte> span, Type type, out Enum value)
         {
-            string enumString = JsonConverter.s_utf8Encoding.GetString(span);
+            string enumString = TempExtensionMethods.TranscodeHelper(span);
+#if BUILDING_INBOX_LIBRARY
```

Comment:
Move the variable into the conditional compilation block. Otherwise, it's unused.